fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -48,10 +48,10 @@ def self.run(params)
 
         options_to_client = lambda do |options|
           options.map do |option|
-            case option
-            when :shake, :anonymous
```

Comment:
Having symbols in the `where` block is actually more beautiful than the strings. I'd probably do it the other way around and do `option.to_sym` and then have `:shake` and `:anonymous` there.